index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```

Comment:
This is a way to suggest changes changed, I like this format because it is easy and allows others to comment on your work.